lib/student.ex

```diff
@@ -34,6 +79,7 @@ defmodule Student do
 
   def name(_student), do: {:error, :not_student}
 
+  @spec projects(any) :: {:error, :noproc | :not_student} | {:ok, list(Project.t())}
```

Comment:
```suggestion
  @spec projects(any()) :: {:error, :noproc | :not_student} | {:ok, list(Project.t())}
```
Ref: https://hexdocs.pm/elixir/typespecs.html#basic-types